.codecov.yml

```diff
@@ -26,6 +26,9 @@ coverage:
   range: 100..100
   status:
     # require patches to be 100%
+    project:
+      default:
+        target: 100%
```

Comment:
FTR with #3162, there's going to be a need to split the checks into MyPy vs. pytest.